felix/proto/felixbackend.proto

```diff
@@ -617,6 +619,7 @@ message RouteUpdate {
   bool local_workload = 9;
   TunnelType tunnel_type = 10;
   bool borrowed = 11;
+  repeated RouteType types = 12;
```

Comment:
I'd move this to line 610.  The id number will be out of order, but that's kindof the reason to have the id number: so that the order in the file can be "natural" while assigning unique IDs as we go.